kuma/wiki/views/create.py

```diff
@@ -30,6 +32,23 @@ def create(request):
     # Try to head off disallowed Template:* creation, right off the bat
     if not Document.objects.allows_add_by(request.user, initial_slug):
         raise PermissionDenied
+    # TODO: Integrate this into a new exception-handling middleware
+    if not request.user.has_perm('wiki.add_document'):
+        email_url = (
+            'mailto:%s?'
+            'subject=Page Creation permission&'
+            'body=My username is %s.'
+            ' I want the page creation permission on %s because...'
+        ) % (config.EMAIL_LIST_MDN_ADMINS,
+             request.user.username,
+             request.get_host())
+        context = {
```

Comment:
Thanks for the clarification. I have worked on the past project where we made some different approach to insert the ``mailto`` link. From that experience, I have made the correction and make a commit
https://github.com/mozilla/kuma/commit/1a02e121f6c401481f7324df91527f10034bfffe

@jwhitlock If you like the approach i have made, you can cherry-pick it in. Let me know your observation. As I dont have development environment running, I can not have a test, but theoretically, it should work.